pkg/clusteragent/autoscaling/workload/model/metadata.go

```diff
@@ -0,0 +1,40 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+//go:build kubeapiserver
+
+package model
+
+import "encoding/json"
+
+// exported for testing purposes
+const (
+	AnnotationsURLKey         = "autoscaling.datadoghq.com/url"
+	AnnotationsFallbackURLKey = "autoscaling.datadoghq.com/fallback-url"
+	AnnotationsSettingsKey    = "autoscaling.datadoghq.com/settings"
+)
+
+// Annotations represents the relevant annotations on a DatadogPodAutoscaler object
+type Annotations struct {
```

Comment:
I don't think the fact that the info comes from annotations is important, the info itself is important. I would see it as a `RecommenderConfiguration` which happens to be from annotations in DPA parsing only.

That could directly be a field in DPAInt `customRecommenderConfiguration *recommenderConfiguration` (in which case the file may not be necessary)